client/file-browser/browse-files.css

```diff
@@ -78,9 +82,11 @@
 .filesScrollableView {
 }
 
-.fileList {
-  padding-top: 8px;
-  padding-bottom: 8px;
+.fileList,
+.fileList > div {
```

Comment:
Generally you probably want to avoid selectors like this if you can help it, better to give the things a unique class (I realize you might not be able to since this is controlled by the virtualized list?)

Reply:
Yeah, this is a `div` inside `react-virtualized`which you can't attach class to, only inline style. And you can't select pseudo classes with inline style :/